explore/src/main/scala/explore/config/sequence/SequenceEditorTile.scala

```diff
@@ -3,31 +3,17 @@
 
 package explore.config.sequence
 
-import crystal.*
-import crystal.react.View
 import eu.timepit.refined.types.string.NonEmptyString
 import explore.components.Tile
 import explore.model.ObsTabTilesIds
 import japgolly.scalajs.react.*
 import japgolly.scalajs.react.vdom.html_<^.*
-import lucuma.core.enums.ObserveClass
-import lucuma.core.math.SignalToNoise
-import lucuma.core.model.Observation
-import lucuma.core.model.Program
-import lucuma.core.model.Target
-import lucuma.ui.syntax.all.given
 
 object SequenceEditorTile:
 
-  def sequenceTile(
-    programId:       Program.Id,
-    obsId:           Observation.Id,
-    targetIds:       List[Target.Id],
-    snPerClass:      Map[ObserveClass, SignalToNoise],
-    sequenceChanged: View[Pot[Unit]]
-  ) =
+  def sequenceTile =
```

Comment:
Can this be a `val` now?

Reply:
Sure. Don't think it makes much of a difference, but it is now a val